ChrootBuild.sh

```diff
@@ -166,84 +166,107 @@ case "${FIPSDISABLE}" in
       ;;
 esac
 
-# Install main RPM-groups
-# shellcheck disable=SC2046
-${YUMDO} @core -- \
-    $(rpm --qf '%{name}\n' -qf /etc/yum.repos.d/* 2>&1 | grep -v "not owned" | sort -u) \
-    authconfig \
-    chrony \
-    cloud-init \
-    cloud-utils-growpart \
-    dracut-config-generic \
-    "${FIPSRPM}" \
-    dracut-norescue \
-    gdisk \
-    grub2 \
-    grub2-tools \
-    iptables-services \
-    iptables-utils \
-    kernel \
-    kexec-tools \
-    lvm2 \
-    ntp \
-    ntpdate \
-    openssh-clients \
-    openssh-server \
-    rootfiles \
-    rsync \
-    selinux-policy-targeted \
-    sudo \
-    tar \
-    vim-common \
-    wget \
-    yum-utils \
-    -abrt \
-    -abrt-addon-ccpp \
-    -abrt-addon-kerneloops \
-    -abrt-addon-python \
-    -abrt-cli \
-    -abrt-libs \
-    -aic94xx-firmware \
-    -alsa-firmware \
-    -alsa-lib \
-    -alsa-tools-firmware \
-    -biosdevname \
-    -gcc-gfortran \
-    -iprutils \
-    -ivtv-firmware \
-    -iwl1000-firmware \
-    -iwl100-firmware \
-    -iwl105-firmware \
-    -iwl135-firmware \
-    -iwl2000-firmware \
-    -iwl2030-firmware \
-    -iwl3160-firmware \
-    -iwl3945-firmware \
-    -iwl4965-firmware \
-    -iwl5000-firmware \
-    -iwl5150-firmware \
-    -iwl6000-firmware \
-    -iwl6000g2a-firmware \
-    -iwl6000g2b-firmware \
-    -iwl6050-firmware \
-    -iwl7260-firmware \
-    -libertas-sd8686-firmware \
-    -libertas-sd8787-firmware \
-    -libertas-usb8388-firmware \
-    -libvirt-client \
-    -libvirt-devel \
-    -libvirt-java \
-    -libvirt-java-devel \
-    -nc \
-    -NetworkManager \
-    -plymouth \
+# Setup the "include" package list
+INCLUDE_PKGS=($(yum groupinfo core 2>&1 | sed -n '/Mandatory/,/Optional Packages:/p' | sed -e '/^ [A-Z]/d' -e 's/^[[:space:]]*[-=+[:space:]]//'))
```

Comment:
At first, I saw this line and was like, "wait, did I submit this PR" ...then remembered I'd used different var-names.